pkg/commands/start.go

```diff
@@ -22,16 +22,16 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func Start(address string, udpLocalPort, tcpLocalPort int, IPv6 bool, hosts map[string]string, upstreamServers []string) error {
+func Start(address string, udpLocalPort, tcpLocalPort int, iPv6 bool, hosts map[string]string, upstreamServers []string) error {
```

Comment:
That seems wrong — should probably be `ipv6` or something?

Reply:
yeah, that looks wrong